src/shared/ui/search-auto-complete/search-auto-complete.tsx

```
@@ -0,0 +1,247 @@
+            }}
+            aria-label={rightIconMode === "clear" ? "clear" : "search"}
+          >
+            {rightIconMode === "clear" ? <IconCancel /> : <IconSearch />}
+          </button>
+        </div>
+
+        {/* 드롭다운 */}
+        {isOpen && !disabled && !isLocked && (
+          <div
+            role="listbox"
+            className={[s.list, s.listTopVariant[variant]].join(" ")}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)